cms/navigation/models.py

```diff
@@ -63,4 +90,16 @@ class NavigationSettings(BaseSiteSetting):
         help_text=_("Select the main menu to display on the site."),
     )
 
-    panels: ClassVar[list[str]] = ["main_menu"]
+    footer_menu = models.ForeignKey(
+        FooterMenu,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="+",
+        help_text=_("Select the footer menu to display on the site."),
+    )
+
+    panels: ClassVar[list] = [
+        FieldPanel("main_menu"),
+        FieldPanel("footer_menu"),
+    ]
```

Comment:
```suggestion
    panels: ClassVar[list] = ["main_menu", "footer_menu"]
```

Wagtail 6.4 now supports shorthand for panels - https://docs.wagtail.org/en/stable/releases/6.4.html#shorthand-for-fieldpanel-and-inlinepanel

Reply:
Thank you for the links and code suggestion 🚀 
✅ [1067893](https://github.com/ONSdigital/dis-wagtail/pull/111/commits/1067893aa8ff7870288467daf7a4a8d5b3ba8869) + removed the import [f2dd703](https://github.com/ONSdigital/dis-wagtail/pull/111/commits/f2dd703f644e36a3e3fd40c6dc1e87cba59f30c5)